### PR TITLE
Adddogtodatabase

### PR DESCRIPTION
**Pull request by ACAPP-dev**
Create dog is working along with get all dogs and get dog by id.  I also added get shelter by id as I needed that in order to create a dog.  This is the base functionality so we can start testing most of the pages of our application (except for add to liked dogs and dislike dog).

### CODE REVIEW

**Review by corith — CHANGES_REQUESTED** [review]
Remove .DS_Store from tracking and add to gitignore. It is an apple specific hidden directory that is not needed in the repository.

File: src/main/java/com/Barker/model/Dog.java [hunk 1]
```diff
@@ -61,6 +64,16 @@
 	@Column(name = "is_adopted")
 	private boolean isAdopted;
 	
+	@JsonIgnore
```
Comment (corith) [inline src/main/java/com/Barker/model/Dog.java:67]: Careful using JsonIgnore, I believe spring boot relies on object mapper when it does certain "magical" things and  when I tried using it to ignore the password field from being returned on a response object it messed up certain getX methods. Not saying this causes a problem, just a word of caution :)

Comment (ACAPP-dev) [reply src/main/java/com/Barker/model/Dog.java:67]: Yeah, I was getting an infinite loop so I added @JsonIgnore.  If we have problems later we might have to find a different solution.  I did add a getter for shelterId so we get the shelterId in the body of the response.  We can use this to keep track of which shelter the dog belongs to if needed...

Comment (ACAPP-dev) [reply src/main/java/com/Barker/model/Dog.java:67]: .DS_Store was removed.

**Review by corith — APPROVED** [review]
Looks good! :)

**Pull request merged**